### ISSUE TITLE
Rewards gained with Linux version not showed in Windows version

### ISSUE THREAD

**Issue by wcastill0** [Spare-Network/spare-blockchain#59]
I got some rewards while using Linux version, but after installing Windows version and status synced, rewards do not show seems missing. Wallet and Network already fully sinced. Wallet is the same as before.

**Comment by wcastill0** [comment]
Type, To, Date, Status, Amount, Fee:
Incoming
spare1zhmmgrj569p...4d5f (removed some chars due to privacy protection)
06/20/2021 7:26:42 AM
Confirmed at height 27320
0.5
0

Type, To, Date, Status, Amount, Fee:
Incoming
spare1zhmmgrj569p...4d5f (removed some chars due to privacy protection)
06/20/2021 7:26:42 AM
Confirmed at height 27320
3.5
0

**Comment by tiny0little** [comment]
these rewards from previous block chain DB, current DB height ~1319

[#55](https://github.com/Spare-Network/spare-blockchain/issues/55)

**Comment by wcastill0** [comment]
Yes, there is a difference in the height between old and new one, but in a previous twitt Spare mentioned they will re issue previous earnings, how can I apply (fill the form) to get those back?

![imagen](https://user-images.githubusercontent.com/39103572/122775033-58ec0e00-d26f-11eb-96e4-4a65a92ac8ef.png)

**Comment by shigeomi-t** [comment]
I too would like to know when and how the acquired coins will be reissued.
